packages/validator/src/validator.test.ts

```diff
@@ -0,0 +1,991 @@
+import fs from 'fs';
+
+import * as validator from './validator';
+import {
+  extraFieldMessage,
+  formattingMessage,
+  keyFormattingMessage,
+  missingParamMessage,
+  sizeExceededMessage,
+} from './utils/messages';
+import { getPath } from './commands/utils';
+import { error } from './utils/logger';
+import { Log } from './types';
+
+const messages: Log[] = [];
+
+const apiTemplate = JSON.parse(fs.readFileSync(getPath('apiSpecifications.json', messages), 'utf8'));
+const endpointsTemplate = JSON.parse(fs.readFileSync(getPath('endpoints.json', messages), 'utf8'));
+const oisTemplate = JSON.parse(fs.readFileSync(getPath('ois.json', messages), 'utf8'));
+
+describe('validator templates loaded', () => {
+  expect(messages).toStrictEqual([]);
+});
+
+const validAPISpecification = `{
```

Comment:
Nit: consider having some fixtures folder with the contents of the specifications and import them instead of inlining.

Reply:
I dont think it would look good, also it might be difficult to detect the test that is failing, but it is true this file is way too big, I rather split it into two in 6cda8a15af3e479132b7d9d6f9378b8578c8f274